File: support-portal-frontent/src/app/modules/supplier-account/_models/SupplierAccountUpdateDto.ts
export interface SupplierAccountUpdateDto {
    account_id: number;
    account_name: string;
    account_password: string;
    supplier: string;
    ml_profile: string | null;
    protection_settings: string | null;
    site: string;
    allow_purchase: boolean;
    enabled: boolean;
    note: string | null;
}

File: support-portal-backend/src/Lumio.SupportPortal.Services/SupplierAccount/Dtos/SupplierAccountUpdateDto.cs
namespace Lumio.SupportPortal.Services.SupplierAccount
{
    public class SupplierAccountUpdateDto
    {
        public int account_id { get; set; }

        public string account_name { get; set; } = null!;

        public string account_password { get; set; } = null!;

        public string supplier { get; set; } = null!;

        public string? ml_profile { get; set; }

        public string? protection_settings { get; set; }

        public string site { get; set; } = null!;

        public bool allow_purchase { get; set; }

        public bool enabled { get; set; }

        public string? note { get; set; }
    }
}
